site/_posts/2017-01-30-keeping-up-with-the-conferences.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Last Week in ManageIQ: Keeping up with the conferences"
+author: Dávid Halász
```

Comment:
Might be better to use skateman here, as you did for your previous 2 posts. In addition I think there are plans to link author ids to github accounts.